### PR TITLE
Update dlp-conditions-and-exceptions.md

### PR DESCRIPTION
**Pull request by aamalhot**
Add details for sender address location

### CODE REVIEW

**Comment by aamalhot** [comment]
Please hold for approval

**Comment by aamalhot** [comment]
@chrfox please review and merge

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 1]
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
 
 ### Senders
 
+For conditions and exceptions that examine the sender's address, you can specify where rule looks for the sender's address. The sender address used by Data Loss Prevention (DLP) to evaluate sender based conditions listed below is the Envelope address. Note that Exchange transport rules (ETR) uses the header address as the default sender address location.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
For conditions and exceptions that examine the sender's address, you can specify where the rule looks for the sender's address. The sender address, used by the DLP rule to evaluate the sender-based conditions listed below, is the Envelope address. Note that Exchange transport rules use the Header address as the default sender address location.
```

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 2]
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
...
 <br>
 
+>[!Note]
+>Starting 20th January 2021, the default sender address location will be moved to the Header address along with the availability of the -SenderAddressLocation parameter to configure desired behavior at a DLP rule level.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
> [!NOTE]
> Starting January 20, 2021, the default sender address location will be moved to the Header address along with the availability of the -SenderAddressLocation parameter to configure desired behavior at a DLP rule level.
```

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 3]
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
...
+
+![image](https://user-images.githubusercontent.com/53205984/145942298-6b435ba6-d146-44fe-a1c5-58babeaf8d7a.png)
+
+At the tenant level, you can configure a Sender Address Location to be used across all rules, unless overridden by a single rule. To revert tenant DLP policy configuration to evaluate sender address from Envelope across all rules, you can run the below command-
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
At the tenant level, you can configure a sender address location to be used across all rules, unless overridden by a single rule. To revert tenant DLP policy configuration to evaluate the sender address from the Envelope across all rules, you can run the following command:
```

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 4]
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
...
+
+```PowerShell
+Set-PolicyConfig –SenderAddressLocation Envelope
+```
+
+To configure sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
To configure the sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@aamalhot Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@aamalhot, kindly commit all my suggested changes. You only seem to have committed one suggestion. Many thanks.

**Comment by aamalhot** [comment]
Thanks @JohanFreelancer9. Completed all

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 5]
```diff
@@ -43,8 +43,28 @@ The tables in the following sections describe the conditions and exceptions that
...
+
+To configure the sender address location at a DLP rule level, the parameter is _SenderAddressLocation_. The available values are:
+
+- **Header**: Only examine senders in the message headers (for example, the **From**, **Sender**, or **Reply-To** fields). This is the default value.
+
+- **Envelope**: Only examine senders from the message envelope (the **MAIL FROM** value that was used in the SMTP transmission, which is typically stored in the **Return-Path** field).
+
+- **Header or envelope** (`HeaderOrEnvelope`) Examine senders in the message header and the message envelope.
+<br>
 ****
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
```

**Comment by aamalhot** [comment]
@JohanFreelancer9 updated

**Review by JohanFreelancer9 — APPROVED** [review]
Thank you, @aamalhot.

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by aamalhot** [comment]
Looks good @chrfox . Please proceed with publish

**Pull request merged**